scripts/lv_conf_internal_gen.py

```diff
@@ -57,6 +57,11 @@
     #else
         #include "../../lv_conf.h"                /*Else assume lv_conf.h is next to the lvgl folder*/
     #endif
+    #if !defined(LV_CONF_H) && !defined(LV_CONF_SUPPRESS_DEFINE_CHECK)
+        /* #include will sometimes silently fail when __has_include is used */
+        /* https://gcc.gnu.org/bugzilla/show_bug.cgi?id=80753 */
+        #pragma message("Possible failure to include lv_conf.h, please read the comment in this file if you get errors")
```

Comment:
```suggestion
        #warning "Possible failure to include lv_conf.h, please read the comment in this file if you get errors"
```

We already used `#warning` on a lot of places and there were no issues. However, we haven't used `#pragma message` so far and it seems it has no advantage over `#warning`.

Reply:
The issue is that `-Wpedantic` gives an error that `#warning` is a GCC extension (apparently only `#error` is in the standard?) and this caused the tests to fail before I defined `LV_CONF_SUPPRESS_DEFINE_CHECK`.

Reply:
Oh,  I always thought `#warning` is standard. 

I tested `#pragma message` in XC16 (compiler fir 16 bit PICs) and it worked. So I think we can merge it.